tests/t568-operator.c

```diff
@@ -153,8 +152,8 @@ int main(int argc, char **argv) {
                                 comp_in*num_dofs;
           const CeedScalar assembled_value = assembled[index];
           const CeedScalar assembled_true_value = assembled_true[index];
-          if (fabs(assembled_value - assembled_true_value) >
-              100.*CEED_EPSILON)
+          if (!(fabs(assembled_value - assembled_true_value) <
+                100.*CEED_EPSILON))
```

Comment:
@nbeams Thanks for pointing this out. The fix (wrong order of arguments) was easy. Building with `-fstack-protector` was very useful and something we should add to CI.